doc/cc65.sgml

```diff
@@ -807,6 +807,10 @@ This cc65 version has some extensions to the ISO C standard.
         <tt/_Static_assert/ is also available as the macro <tt/static_assert/ in
         <tt/assert.h/.
 
+<item>  cc65 supports bit-fields of any integral type that is int-sized or
+        smaller, and enum types with such types as their underlying type.
+        (Only <tt/int/, <tt/signed int/, and <tt/unsigned int/ are required.)
```

Comment:
Perhaps a small clarification should be added here: `"... are required by the standard."`.

Reply:
I think "enum types with such types as their underlying type" as documented here should be tested.

Reply:
> I think "enum types with such types as their underlying type" as documented here should be tested.

Yes, these tests have been missing for years. 

Enum bit-fields apparently used to work: https://godbolt.org/z/KnEbcq

(Godbolt is using f95481f from 1.5 years ago as of when I made that link.)

Probably more tests for different underlying types should have been added when you did your enum work.

It will take a little work to fix the enum bit-fields, so I have just deleted mention of enum bit-fields for now. It's orthogonal to this PR anyway.

Reply:
I wasn't aware of its missing from the tests either. It appeared to be working (aside from #1047) when I modified the check on bit-field type size. So I thought it was all right.

Anyways, are you going to work on fixing enum bit-fields?

Reply:
I appear to have broken it, so I'll try to fix it.

Reply:
> I appear to have broken it, so I'll try to fix it.

Do you see that fix part of this PR? Or do you consider this PR as ready to be merged?

Reply:
Better to fix #1244 first, separately.

Reply:
https://github.com/cc65/cc65/issues/1244 is closed by now...